CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.0.10 October 26, 2024
+
+Added `intsToRanges` and `intsToStringRanges` to `dark/utils.py`. Ran `black`.
```

Comment:
Why mention that you ran `black` in the changelog? In case it broke something?

Reply:
Ah, I was just copying my commit message. I'll remove it.